.github/workflows/beaker.yml

```diff
@@ -148,6 +148,11 @@ jobs:
     name: "${{ matrix.name }}"
     steps:
       - uses: actions/checkout@v4
+      - name: Setup podman
```

Comment:
After https://github.com/voxpupuli/gha-puppet/pull/49 this needs a conditional on the beaker hypervisor so we don't do this for vagrant.